tests/integration/it_auth_test.py

```diff
@@ -63,12 +63,23 @@ def test_activate(client):
     assert "Successfully activated your account." not in decoded_response
 
 
-"""
-    @todo #388:30min Forgot password routine. Test a forgot password logic: 
-     create an random password, Login with it, see it works.
-     change it for the user with the received e-mail, find it changed.
-"""
-
+def test_forgot_password_routine(db_session, client):
+    employee = factories.EmployeeFactory(
+        username="uname",
+        password=auth_hash("pass"),
+        email="mail@mail.com"
+    )
+    db_session.add(employee)
```

Comment:
@ammaratef45 Factory already commited `employee`, so you dont need to use `db_session` directly.